crates/client/rpc-core/src/utils.rs

```diff
@@ -56,7 +56,9 @@ pub fn to_invoke_tx(tx: BroadcastedInvokeTransaction) -> Result<InvokeTransactio
                 invoke_tx_v1.signature.iter().map(|x| (*x).into()).collect::<Vec<Felt252Wrapper>>(),
             )
             .map_err(|e| anyhow!("failed to convert signature: {:?}", e))?,
-            sender_address: invoke_tx_v1.sender_address.to_bytes_be().into(),
+
+            // Safe to unwrap, starknet-core already parsed the FieldElement with jsonrpsee.
+            sender_address: Felt252Wrapper::try_from(&invoke_tx_v1.sender_address.to_bytes_be()).unwrap(),
```

Comment:
there is a `impl From<FieldElement> for Felt252Wrapper` so why not just

```
sender_address: invoke_tx_v1.sender_address.into()
```

Reply:
@ClementWalter it's a very nice catch and in the overall refacto I'll double check I didn't miss something like this.
As before it was `H256/U256` the conversion to bytes was needed. Now it's not, so it's a useless call to `to_bytes_be`, which can then be refact as you said. 👍

Reply:
> there is a `impl From<FieldElement> for Felt252Wrapper` so why not just
> 
> ```
> sender_address: invoke_tx_v1.sender_address.into()
> ```

I find this way, more cumbersome for sure but clearer aswell. Note that in this case it would actually be `.try_into().unwrap`

Reply:
I think that we should favor everywhere `(try_)into()` instead of `Type::(Try)From` because the latter is redundant and heavier to maintain. For example, it's possible that we later change the function signature and update if necessary the input type with a new `From`, but we won't have to rewrite everything. It seems to me that `into` is more versatile and more idiomatic. The compiler knows what to do, we don't need to write it as well